### PR TITLE
Slayer changes

### PR DESCRIPTION
**Pull request by X0-11**
Fixes weapons. #310
Adds spartan helm light
Adds shields to slayer spartan suits
Makes needler more effective at embedding
Adds medical dispensing system to slayer spartan suits.
Lowers pain from biofoam application.
Auto-allows respawn for all ghosts every 30 seconds. #312
Add a simple shield level display

### CODE REVIEW

**Comment by X0-11** [comment]
Hm. Merge confilcts with the changelog.
EDIT: Should be ready for review now.

File: code/modules/halo/gamemodes/slayer-team-slayer/mode_slayer.dm [hunk 1]
```diff
@@ -23,6 +27,23 @@
 		return 1
 	return 0
 
+/datum/game_mode/slayer/process() //Used to allow respawns after few minutes. Also auto-kills people after a threshold.
+	for(var/mob/living/carbon/human/i in GLOB.player_list)
```
Comment (caelaislinn) [inline code/modules/halo/gamemodes/slayer-team-slayer/mode_slayer.dm]: Aargh this is a bad thing to put in a process() tick. Did you look at mucking around with life or creating a new species?

**Review by caelaislinn — CHANGES_REQUESTED** [review]
How did you go with the on death runtime? I can have a look at that one for you if you like

File: code/modules/halo/gamemodes/slayer-team-slayer/mode_slayer.dm (hunk 1 above)
Comment (X0-11) [reply code/modules/halo/gamemodes/slayer-team-slayer/mode_slayer.dm]: Perhaps sleep() it? I'll look into what could be used instead.

**Review by caelaislinn — APPROVED** [review]

**Comment by X0-11** [comment]
I have no idea what's causing the fail/runtime. Tried making it return 1, still fails/runtimes.

**Comment by X0-11** [comment]
Besides the runtime which was present before my changes, this should be ready to merge.

**Comment by X0-11** [comment]
Can't believe I didn't check that. Should be ready for merge now then

**Pull request merged**